src/writer/vote.ts

```diff
@@ -113,7 +104,31 @@ export async function action(body, ipfs, receipt, id): Promise<void> {
     }
   }
 
-  //
+  // Get proposal
+  const proposal = await getProposal(msg.space, msg.payload.proposal);
+
+  // Store voting power if no delegation strategies
+  let withDelegation = false;
+  const strategies = jsonParse(proposal.strategies);
+  const names = strategies.map(strategy => strategy.name);
+  names.forEach(name => {
+    if (name.includes('delegation')) withDelegation = true;
```

Comment:
Should we also add `erc20-with-delegation`?

Reply:
This already check for erc20-with-delegation

Reply:
@ChaituVR anything that `includes` `'delegations'.`

Reply:
Oh!!! got it. how did I miss that.. thanks, @mktcode :D

Reply:
Maybe the code should be [shortened a bit](https://github.com/snapshot-labs/snapshot-hub/pull/301#discussion_r851613946). :D